### PR TITLE
[Docs] Fix some links in latest docs

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

File: docs/content/latest/explore/ysql-language-features/pg-extensions.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ PostgreSQL provides a way to extend the functionality of a database by bundling
 This page describes the PostgreSQL extensions supported by YugabyteDB. These extensions can either be pre-bundled or may require installation:
 
  
-* **Pre-bundled extensions:** These are extensions that are included in the standard YugabyteDB distribution and can be enabled in YSQL by running the [`CREATE EXTENSION`](../the-sql-language/statements/ddl_create_extension) statement.
-* **Extensions requiring installation:** Other extensions have to be installed manually before they can be enabled with the [`CREATE EXTENSION`](../the-sql-language/statements/ddl_create_extension) statement. Refer [how to install an extension?](#installing-an-extension)
+* **Pre-bundled extensions:** These are extensions that are included in the standard YugabyteDB distribution and can be enabled in YSQL by running the [`CREATE EXTENSION`](../../../api/ysql/the-sql-language/statements/ddl_create_extension) statement.
+* **Extensions requiring installation:** Other extensions have to be installed manually before they can be enabled with the [`CREATE EXTENSION`](../../../api/ysql/the-sql-language/statements/ddl_create_extension) statement. Refer [how to install an extension?](#installing-an-extension)
```
Comment (polarweasel) [inline docs/content/latest/explore/ysql-language-features/pg-extensions.md:22]: While we're at it, let's avoid a redirect :)
```suggestion
* **Pre-bundled extensions:** These are extensions that are included in the standard YugabyteDB distribution and can be enabled in YSQL by running the [`CREATE EXTENSION`](../../../api/ysql/the-sql-language/statements/ddl_create_extension/) statement.
* **Extensions requiring installation:** Other extensions have to be installed manually before they can be enabled with the [`CREATE EXTENSION`](../../../api/ysql/the-sql-language/statements/ddl_create_extension/) statement. Refer to [how to install an extension](#installing-an-extension).
```

File: docs/content/latest/explore/ysql-language-features/sql-feature-support.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Altering tables | ✓ | [`ALTER TABLE`  documentation](https://docs.yugabyte.com
 
  Component | Supported |  Documentation|
 -----------|-----------|---------
- Indexes | ✓ | [Indexes documentation](https://docs.yugabyte.com/latest/explore/indexes-constraints/overview/)|
+ Indexes | ✓ | [Indexes documentation](https://docs.yugabyte.com/latest/explore/indexes-constraints/)|
```
Comment (polarweasel) [inline docs/content/latest/explore/ysql-language-features/sql-feature-support.md:79]: Ugh why aren't these all relative links??? (Answer: this was from a PR that didn't get adequately reviewed... I'll catch them all sometime.)

**Review by polarweasel — APPROVED** [review]
Thanks, @def-! Left one change, merge when done. :)

File: docs/content/latest/explore/ysql-language-features/sql-feature-support.md (hunk 2 above)
Comment (def-) [reply docs/content/latest/explore/ysql-language-features/sql-feature-support.md:79]: I was also wondering about this, but didn't want to just change it.

**Pull request merged**